ls1intum/Artemis
# `General`: Fix displaying of emojis

### Checklist
#### General


#### Server
#### Client


### Motivation and Context


Fixes #6642 

### Description

The PR updates the emoji sheet to the one used by the latest version of `@ctrl/ngx-emoji-mart`

### Steps for Testing

Prerequisites:
- 1 Course with communication feature enabled
- 1 Instructor
- 1 Exercise

1. Go to any exercise
2. Create a post
3. React to the post
4. Confirm that the emojis are working again (dark & light mode)
5. Also test that our custom emojis (🔝, 🔙, ™️, etc.) are shown correctly in dark mode

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage




Unchanged

### Screenshots
